docs/modules/reference/pages/configuration/minion-confd/test-confd.adoc

```diff
@@ -0,0 +1,28 @@
+
+[[confd-template-test]]
+= Test/develop confd templates
```

Comment:
Personally not a fan of this title.  I would prefer either "Test" or "Develop" but not both.

Reply:
Good point. Will update.